backup/idm/keycloak/dBildungscloud-realm.json

```diff
@@ -1010,7 +1010,7 @@
     "xContentTypeOptions" : "nosniff",
     "xRobotsTag" : "none",
     "xFrameOptions" : "SAMEORIGIN",
-    "contentSecurityPolicy" : "frame-src 'self'; frame-ancestors 'self'; object-src 'none';",
+    "contentSecurityPolicy" : "frame-src 'self' data:; frame-ancestors 'self'; object-src 'none';",
```

Comment:
@pkleybolte can you please review this one line?

Reply:
how can this be relevant to edusharing stuff at all? at this place? can't be correct

Reply:
We forget to delete this. Delete it in new commit.